astropy/cosmology/_src/flrw/base.py

```diff
@@ -94,7 +95,14 @@ def quad(*args, **kwargs):
 
 
 @dataclass_decorator
-class FLRW(Cosmology, ScaleFactor, TemperatureCMB, _CriticalDensity, _BaryonComponent):
```

Comment:
the `def Om()` method should be removed from here, so that it uses the one on `_MatterDensity`.

Reply:
That was an oversight. Thanks for catching that. Its done.